observability/logging/logging_alerts/default-logging-alerts.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Logging alerts are installed as part of the {clo} installation. Alerts depend on metrics exported by the log collection and log storage backends. These metrics are enabled if you selected the option to *Enable operator recommended cluster monitoring on this namespace* when installing the {clo}. For more information about installing logging Operators, see xref:../../logging/cluster-logging-deploying#cluster-logging-deploy-console_cluster-logging-deploying[Installing {logging} using the web console].
+Logging alerts are installed as part of the {clo} installation. Alerts depend on metrics exported by the log collection and log storage backends. These metrics are enabled if you selected the option to *Enable operator recommended cluster monitoring on this namespace* when installing the {clo}. For more information about installing logging Operators, see xref:../../../observability/logging/cluster-logging-deploying#cluster-logging-deploy-console_cluster-logging-deploying[Installing {logging} using the web console].
```

Comment:
Actually links to section titled "Installing the Red Hat OpenShift Logging Operator by using the web console"

Reply:
Please see my https://github.com/openshift/openshift-docs/pull/74819#discussion_r1577387726.